src/context.ts

```diff
@@ -65,6 +65,7 @@ export interface ContextValue {
   cache?: CacheConfiguration;
   subgraph?: SubgraphConfiguration;
   partnerId?: string;
+  locale?: string;
```

Comment:
Could we be a bit more restrictive here and have string literal types for this prop?

As in, having a `LocaleType`
```typescript
type LocaleType = "en" | "es" | "fr"; // etc
```

Reply:
sure! I'll find out which locales we support

Reply:
fixed on d53eb2fc1b404ad58a1f6df757b73691fbf57a05